chatmaild/src/chatmaild/user.py

```diff
@@ -58,7 +58,8 @@ def set_password(self, enc_password):
             if not self.addr.startswith("echo@"):
                 logging.error(f"could not write password for: {self.addr}")
                 raise
-        self.enforce_E2EE_path.touch()
+        if not self.addr.startswith("echo@"):
+            self.enforce_E2EE_path.touch()
```

Comment:
would rather do this in echo-bot setup (chatmaild/echo.py) not in this generaly code path

Reply:
echo.py is run by the `echobot` user, which doesn't have write permission in this path... we could do stuff with `setfacl` I guess, as this is anyway installed to get echobot running, but that sounds overly complicated.